common/test/acceptance/tests/studio/test_studio_course_team.py

```diff
@@ -26,6 +26,21 @@ def _make_user(self, username):
         ).visit()
         return user
 
+    def _update_user(self, **update_values):
+        """
+        Update other_user with provided values
+
+        Arguments:
+            `update_values`: dictionary containing values of attributes to be updated
+        """
+        AutoAuthPage(
+            self.browser, no_login=True, username=self.other_user.get('username'),
+            email=self.other_user.get('email'), password=self.other_user.get('password'),
+            is_active=update_values.get('is_active')
+        ).visit()
+
+        self._go_to_course_team_page()
```

Comment:
The name of the method is saying `_update_user` but it is also `self._go_to_course_team_page()` which doesnt fit with the user name. I mean `self._go_to_course_team_page()` doesn't blong in this method.